packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -292,9 +286,9 @@ class _CupertinoButtonState extends State<CupertinoButton> with SingleTickerProv
     final Color effectiveFocusOutlineColor = widget.focusColor ??
       HSLColor
         .fromColor((backgroundColor ?? CupertinoColors.activeBlue)
-        .withOpacity(_kCupertinoFocusColorOpacity))
-        .withLightness(_kCupertinoFocusColorBrightness)
-        .withSaturation(_kCupertinoFocusColorSaturation)
+        .withOpacity(kCupertinoFocusColorOpacity))
+        .withLightness(kCupertinoFocusColorBrightness)
+        .withSaturation(kCupertinoFocusColorSaturation)
```

Comment:
It looks a little odd that it is all in a line when it's not directly chaining. Maybe?
```
HSLColor
        .fromColor((backgroundColor ?? CupertinoColors.activeBlue)
          .withOpacity(kCupertinoFocusColorOpacity)
        ).withLightness(kCupertinoFocusColorBrightness)
        .withSaturation(kCupertinoFocusColorSaturation)
```
or something?

Reply:
That's true. The cascade does look nice though :)